lib/backend/k8s/syncer.go

```
@@ -668,9 +678,11 @@ func (syn *kubeSyncer) performSnapshot(versions *resourceVersions) (map[string][
 
 			versions.networkPolicyVersion = npList.ListMeta.ResourceVersion
 			for _, np := range npList.Items {
-				pol, _ := syn.converter.NetworkPolicyToPolicy(&np)
-				snap[KEY_NP] = append(snap[KEY_NP], *pol)
-				keys[KEY_NP][pol.Key.String()] = true
+				pol, err := syn.converter.NetworkPolicyToPolicy(&np)
+				if err == nil {
+					snap[KEY_NP] = append(snap[KEY_NP], *pol)
+					keys[KEY_NP][pol.Key.String()] = true
+				}
```

Can we have an error log here too please since we're ignoring the error.